src/Elevator.java

```diff
@@ -1,4 +1,44 @@
+import java.lang.Math;
+import java.time.LocalTime;
+import java.util.ArrayList;
 
-public class Elevator {
+public class Elevator implements Runnable {
+  private int currentFloor;
+  private boolean isStationary;
+  private Middleman middleman;
+  private Direction direction;
+  private ArrayList<ElevatorButton> buttons;
 
+  public Elevator(Middleman middleman) {
+    this.middleman = middleman;
+    isStationary = true;
+    currentFloor = 0;
+    direction = Direction.UP;
+    buttons = new ArrayList<ElevatorButton>();
```

Comment:
the constructor should also initialize all of the elevator button objects and add them to the buttons arraylist